src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        int index = new Random().nextInt(colors.length);
```

Comment:
you're creating new Random object each time the method is called. Please declare Random object as a class field and reuse it